torch/distributed/fsdp/_exec_order_utils.py

```diff
@@ -79,7 +79,7 @@ def init(
             index = len(self.all_handles)
             self.all_handles.append(handle)
             handle._handle_index = index
-            self.param_to_fqn = _get_param_to_fqns(root_module)
```

Comment:
This does not need to be in a loop `for handle in traversal_utils._get_fsdp_handles(root_module):`. The previous PR accidentally indented this.